websites/G/Gunivers/presence.ts

```diff
@@ -87,12 +94,24 @@ presence.on("UpdateData", async () => {
     presenceData.smallImageKey = "workspace";
     presenceData.details = "Viewing a page:";
     presenceData.state = "Gunivers Workspace";
+  if(button) presenceData.buttons = [
+      {
+        label: "Visit Website",
+        url: "https://project.gunivers.net/"
+      }
+    ]
```

Comment:
![image](https://user-images.githubusercontent.com/9348108/108812770-009e4b80-75b0-11eb-816d-658dc1a5af00.png)
If you want to keep this button, change label for it